source/collections/VectorList.ooc

```diff
@@ -74,7 +74,7 @@ VectorList: class <T> {
 	}
 	pop: func -> T {
 		result := this[0]
-		this remove(0)
+		this removeAt(0)
```

Comment:
The second example was my thought but the first example was cleaner. Don't know if that will leak or not. I'll check that out.